src/governance.rst

```diff
@@ -144,6 +148,25 @@ the time when a vote is called, but may never be less than 24 hrs.
     * Sensitive
     * 75% Majority to pass
 
+:Overall workflow and packaging policies: The proposer can choose to
+    create a poll with an external tool or call
+    for voting on the GH issue in question.
+    The voting must period be open for at least one core
```

Comment:
`The voting must period be open...` -> `The voting period must be open...`

Reply:
Thanks!